catalog-info.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: sda-dropwizard-commons
+  annotations:
+    github.com/project-slug: 'sda-se/sda-dropwizard-commons'
+    fossa.io/project-name: sda-dropwizard-commons
+    sonarqube.org/project-key: SDA-SE_sda-dropwizard-commons
+  description: |
+    SDA Dropwizard Commons is a set of libraries to bootstrap services easily that follow
+    the patterns and specifications promoted by the SDA SE.
```

I would prefer to clarify the backend aspect and to name the technology. Is Markdown allowed?
Then something like this would be nice:
```suggestion
    SDA Dropwizard Commons is a set of libraries to bootstrap Java backend services easily.
    It configures [Dropwizard](https://www.dropwizard.io/) applications to follow the patterns 
    and specifications promoted by the SDA SE.
```

---

That field doesn't support markdown and should be a short description. Maybe we should make it even shorter.